client/src/app/shared/shared-instance/instance.service.ts

```diff
@@ -45,6 +45,9 @@ export class InstanceService {
       codeOfConduct: '',
       moderationInformation: '',
       administrator: '',
+      creationReason: '',
+      maintenanceLifetime: '',
+      businessModel: '',
```

Comment:
You need to add these here too :

 https://github.com/Chocobozzz/PeerTube/blob/4a7f90248b546803cab07188521ac97e175b31e6/client/src/app/%2Babout/about-instance/about-instance.component.ts#L21

Reply:
Done. That was somehow fixing a compilation error on my end, which I didn't encounter yesterday. So that's two birds with one stone !
Thanks 🙂